odk/odk.py

```diff
@@ -515,6 +518,9 @@ class SSSOMMappingSetGroup(JsonSchemaMixin):
 
     release_mappings : bool = False
     """If set to True, mappings are copied to the release directory."""
+
+    mapping_extractor : str = "sssom-py"
+    """The tool to use to extract mappings from an ontology ('sssom-py' or 'robot')."""
```

Comment:
Maybe to be more future proof this could be sssom-py and sssom-java (in case someone implements a stellar mapping extraction method in robot itself :P). Ok fantasising now, but.. suggestion.

Reply:
> in case someone implements a stellar mapping extraction method in robot itself

I think that’s _highly_ unlikely :D but OK, I have no objection with that.